src/components/Core/ParallaxCaseStudies/index.tsx

```diff
@@ -0,0 +1,12 @@
+import ParallaxUseCasesElement from '@/components/Core/ParallaxCaseStudies/ParallaxUseCasesElement'
+import ParallaxText, { type ParallaxTextProps } from '@/components/common/ParallaxTextOld'
+
+const ParallaxCaseStudies = (props: ParallaxTextProps) => {
+  return (
+    <ParallaxText {...props} variant="image-text">
+      <ParallaxUseCasesElement items={props.items ?? []} />
```

Comment:
If there are no `props.items`, this will still render the wrapper and `FrameImage`.
```suggestion
      {props.item ? <ParallaxUseCasesElement items={props.items} /> : null}
```